src/main/kotlin/org/codefreak/codefreak/service/AnswerService.kt

```diff
@@ -57,9 +58,13 @@ class AnswerService : BaseService() {
   @Transactional
   fun deleteAnswer(answerId: UUID) = answerRepository.deleteById(answerId)
 
+  @Transactional
   fun setFiles(answer: Answer): OutputStream {
     require(answer.isEditable) { "The answer is not editable anymore" }
-    return fileService.writeCollectionTar(answer.id).afterClose { containerService.answerFilesUpdated(answer.id) }
+    return fileService.writeCollectionTar(answer.id).afterClose {
+      answer.updatedAt = Instant.now()
```

Comment:
save?

Reply:
Should happen automatically b/c of `@Transactional`

Reply:
?

Reply:
`answer` is a managed entity and will be automatically saved after leaving the method

Reply:
okay 😅